docs/source/conf.py

```diff
@@ -104,56 +104,43 @@ def build_api_docs(out_dir):
     docs = osp.join(HERE, os.pardir)
     root = osp.join(docs, os.pardir)
     docs_api = osp.join(docs, "source", "api")
-    api_index = osp.join(docs_api, "algorithm", "index.html")
-
-    if osp.exists(api_index):
-        # avoid rebuilding docs because it takes forever
-        # `make clean` to force a rebuild
-        print(f"already have {api_index}")
-    else:
-        print("Building lumino API docs")
-        npm = [shutil.which('npm')]
-        check_call(npm + ['install', '-g', 'yarn'], cwd=root)
-        yarn = [shutil.which('yarn')]
-        check_call(yarn, cwd=root)
-        check_call(yarn + ["docs"], cwd=root)
+
+    print("Building lumino API docs")
+    npm = [shutil.which('npm')]
+    check_call(npm + ['install', '-g', 'yarn'], cwd=root)
+    yarn = [shutil.which('yarn')]
+    check_call(yarn, cwd=root)
+    check_call(yarn + ["docs"], cwd=root)
 
     dest_dir = osp.join(out_dir, "api")
     print(f"Copying {docs_api} -> {dest_dir}")
     if osp.exists(dest_dir):
         shutil.rmtree(dest_dir)
     shutil.copytree(docs_api, dest_dir)
-    shutil.copy(osp.join(HERE, 'api_index.html'), osp.join(dest_dir, 'index.html'))
```

Comment:
If we set `readme` to `none` in the Typedoc config file then we don't need to do this copy step.